app/views/layouts/admin/_search.html.erb

```diff
@@ -0,0 +1,14 @@
+<form class="form-inline mr-auto ml-md-3 my-2 my-md-0 mw-100 navbar-search">
```

Comment:
I don't think I will have a search bar for now.
![image](https://user-images.githubusercontent.com/8007722/72658362-e130fa00-397d-11ea-8e82-fa0c20a2bd83.png)

I think the search could be replaced by the latest notification alert, should be kind of small:
![image](https://user-images.githubusercontent.com/8007722/72658367-0160b900-397e-11ea-9385-3820977ec0a9.png)

Reply:
![image](https://user-images.githubusercontent.com/8007722/72821257-4581e100-3c3e-11ea-9562-a7ad001a13d4.png)